bash/build_utils.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#
```

Comment:
Breaking this out into a utils is nice but typically we have a `build.sh` file and I don't see that here. Is the `02_preprocessing.sh` meant to replace the `build.sh` script?

Reply:
in a sense yea, I think what's missing is `03_build.sh` to continue the approach of numbered build steps. the github action is the high level "build"

I'll add a file or two to show a more complete but still placeholder file structure